app/views/invoices/_form.html.haml

```diff
@@ -82,11 +97,27 @@
               = label_tag @invoice.client.contact_details.first.email
           .box-footer
             = hidden_field_tag :email_send, :value => "false", :id => "email_send"
-            = f.submit "Save as Draft", class: "btn btn-primary", id: "submit_button", data: { disable_with: "Saving..." }
+            = f.button "Save as Draft", class: "btn btn-primary", id: "submit_button", data: { disable_with: "Saving..." }
             = f.button "Save & Send by Email", class: "btn btn-primary", id: "send_button", data: { disable_with: "Saving..." }
 
 :javascript
+  $(document).ready(function() {
+
+    $('#invoice_date_of_issue').datepicker({autoclose: true});
+    $('#invoice_paid_to_date').datepicker({autoclose: true});
```

Comment:
you can combine binding of both like `$('#invoice_date_of_issue, #invoice_paid_to_date').datepicker({autoclose: true});` in a single expression.